JustSaying.Tools/JustSaying.Tools.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\build\common.props" />
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="App.config" />
```

Comment:
No, I am full of lies today. The binding redirects are auto generated by the sdk import in the new world, in fact this was conflicting with the `app.config` ones. I've deleted them 👍